### DISCUSSION TITLE
Colors have changed in 3.0.19

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#160] [Q&A] [answered]
Can it be that the map color has changed in 3.0.19? I am not sure yet if this is good or bad for navigation. I just recognized it.

**Comment by devemux86** [comment]
@DolomitesTom do you mean the colors of the map theme?
The map themes are the same. What map theme?

Or do you mean something else?
Can you post a screenshot?

**Reply by DolomitesTom** [reply]
Have already updated all my devices... I just had the impression that with the map theme biker the colors are a bit darker than in 3.0.18. Maybe I am wrong. Cannot compare this any more. In case of doubt, it is not a problem.

**Reply by devemux86** [reply]
@DolomitesTom also check if you changed the "Settings | Map | Appearance | Map color".

**Reply by DolomitesTom** [reply]
still "Normal" as before

**Comment by WolfgangZ42** [comment, marked as answer]
Here the sceenshot of "Biker" (not my usually used theme):
<img width="200" src="https://github.com/devemux86/cruiser/assets/121806238/b2e05726-1593-4d60-8822-ac5442d797a4"/>    <img width="200" src="https://github.com/devemux86/cruiser/assets/121806238/39036477-e50d-48e7-9c0e-d8168c741e35"/>
----------Cruiser 3.0.18----------------------Cruiser 3.0.19----------
I see no change ...

Establishing back my favorite map design I had some difficulties to find the right one:
<img width="200" src="https://github.com/devemux86/cruiser/assets/121806238/b580d2b3-d976-498c-bc35-71beaee282d8"/>

**Reply by devemux86** [reply]
@WolfgangZ42 are these external map themes?

Is the "default" the xml filename inside the theme zip file?

**Reply by MotoUKRider** [reply]
Perhaps you have my edited biker+dark theme in there, too?  I called them `default.xml` and `dark.xml` so the newer (tidy) list shown above *may* be confusing.  I'll probably change their names the next time I have anything new to publish, but at least you can fix it yourself by opening, unzipping, renaming the files etc :)

**Reply by devemux86** [reply]
There should be a "Default" (with uppercase "D") from the included themes.
The image shows 3 "default" (with lowercase "d") and 1 "dark".

Perhaps we need the parent zip name to better understand such differences?

**Reply by WolfgangZ42** [reply]
> Perhaps we need the parent zip name to better understand such differences?

**That would fix it!** Sure we can rename the files - I thought "default" is prescribed.

**Reply by MotoUKRider** [reply]
I have changed my filenames, so if I need to re-publish my theme again - you'll get a MUK prefix!

![motoukrider-theme-list](https://github.com/devemux86/cruiser/assets/125696109/fe46b4c4-2e4c-4740-b3b4-5b09ae08e73e)

**Reply by WolfgangZ42** [reply]
I tried the same (with MRUK 😉) but it doesn't work - strange! I had to restore my changes.

**Comment by airald** [comment]
Also changed the blue used on panels. With dark theme, is darker than before, and the contrast with dark grey is lower than before.

**Reply by devemux86** [reply]
The only color change in 3.0.19 is the accent color in dark theme:
- 3.0.18 had the same blue color in dark theme as the light theme (not good, as it reduces readability)
- 3.0.19 has again the correct colors: blue on light theme, cyan on dark theme

All other colors can be changed in "Settings | Application | Appearance".

<img src="https://github.com/devemux86/cruiser/assets/3484020/78f5a999-6398-41b0-99a5-5fb57d706ac5" width="50%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/621d49af-0bf7-44a6-a4ea-111629329e69" width="50%"/>

**Reply by airald** [reply]
yes, just now I have the update available. Is working good again with the desired colors in settings. Good job!
